### ISSUE TITLE
Message toolbar does not show anymore with Firefox in v6.8.0

### ISSUE THREAD

**Issue by fangebee** [RocketChat/Rocket.Chat#32461]
### Description:

The message toolbar (allowing to add reactions/quote/reply in thread/...) is not showing anymore in Firefox after upgrading to Rocketchat v6.8.0.
It is still visible in Chrome when hovering a message with the mouse. See the screenshot:

![toolbar_visible_in_chromium](https://github.com/RocketChat/Rocket.Chat/assets/78966083/8a985517-03bd-4768-b1b8-9cbf56db5cd8)

I don't know what happens with other Web browsers.

**Comment by reetp** [comment]
> The message toolbar (allowing to add reactions/quote/reply in thread/...) is not showing anymore in Firefox after upgrading to Rocketchat v6.8.0.

> It is still visible in Chrome when hovering a message with the mouse. See the screenshot:

Please see this and tell us which versions your browsers are.

I just tested on open.rocket.chat with the latest versions of both browsers and it worked OK.

https://github.com/RocketChat/Rocket.Chat/issues/32453

**Comment by dougfabris** [comment]
Hi there, thanks for the contribution! 🚀 💯 

Sorry, this error does not happen on the latest. It was fixed here: #32415.

[Rocket.Chat](https://rocket.chat/) provides support for, at least, 3 months for each release and 6 months for the latest releases of each major version

I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates. 

---

**Questions? Help needed? Feature Requests?** 

- Join our [Open Server](https://open.rocket.chat/) in the #support channel and feel free to raise a question
- Join our [Community Forum](https://forums.rocket.chat/) and search/create a post there

**Comment by fangebee** [comment]
> > The message toolbar (allowing to add reactions/quote/reply in thread/...) is not showing anymore in Firefox after upgrading to Rocketchat v6.8.0.
> 
> > It is still visible in Chrome when hovering a message with the mouse. See the screenshot:
> 
> Please see this and tell us which versions your browsers are.

To my knowledge, it works fine in Debian-based Chromium v125.0.6422.76 but it fails with Debian-based Firefox ESR (v115.11).


> I just tested on open.rocket.chat with the latest versions of both browsers and it worked OK.

Well, all our users have Firefox-ESR as their default browser running on Debian stable computers. Unfortunately, upgrading Firefox is not option for us.


> #32453

Sorry, I didn't find this report. It's clearly a duplicate.


> I would suggest you update your Rocket.Chat version to the latest stable version, which you can find in [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases) and, check it [HERE](https://docs.rocket.chat/resources/get-support/enterprise-support#rocket.chat-versions) the table with versions and End of Life dates.

We already have Rocketchat v6.8.0 with enterprise support. From what I can see, there is no newer version available on [Releases Page](https://github.com/RocketChat/Rocket.Chat/releases).
Can you please point me to the "latest stable version"?

**Comment by fangebee** [comment]
> Sorry, this error does not happen on the latest version. It was fixed here: #32415.

OK, I think I understand the problem now.
Looks like the fix was done in branch develop but didn't reach tag v6.8.0.

Comparing [apps/meteor/package.json](https://github.com/RocketChat/Rocket.Chat/blob/6.8.0/apps/meteor/package.json) from tag 6.80 and [apps/meteor/package.json](https://github.com/RocketChat/Rocket.Chat/blob/develop/apps/meteor/package.json) from branch develop:

```
diff --git a/apps/meteor/package.json b/apps/meteor/package.json
index 127439f74c..cbffa5140b 100644
--- a/apps/meteor/package.json
+++ b/apps/meteor/package.json
@@ -1,7 +1,7 @@
 {
        "name": "@rocket.chat/meteor",
        "description": "The Ultimate Open Source WebChat Platform",
-       "version": "6.9.0-develop",
+       "version": "6.8.0",
        "private": true,
        "author": {
                "name": "Rocket.Chat",
@@ -242,7 +242,7 @@
                "@rocket.chat/favicon": "workspace:^",
                "@rocket.chat/forked-matrix-appservice-bridge": "^4.0.2",
                "@rocket.chat/forked-matrix-bot-sdk": "^0.6.0-beta.3",
-               "@rocket.chat/fuselage": "^0.53.7",
+               "@rocket.chat/fuselage": "^0.53.4",
                "@rocket.chat/fuselage-hooks": "^0.33.1",
                "@rocket.chat/fuselage-polyfills": "~0.31.25",
                "@rocket.chat/fuselage-toastbar": "^0.31.26",
```

Could you please release a tag 6.8.1 including the fix?

**Comment by mboudet** [comment]
Would it be possible to also backport it to 6.7, which is still in support until Oct 2024? We have this issue with 6.7.3, Docker edition.

**Comment by dougfabris** [comment]
@mboudet I added the fix to the 6.7.4 patch release
